apps/trading/components/navbar/navbar.tsx

```diff
@@ -179,6 +185,19 @@ const NavbarMenu = ({ onClick }: { onClick: () => void }) => {
             {t('Portfolio')}
           </NavbarLink>
         </NavbarItem>
+        <NavbarItem>
+          <span
+            className={classNames(
+              'block lg:flex lg:h-full flex-col justify-center',
+              'px-6 py-2 lg:p-0 text-lg lg:text-sm',
+              'hover:text-vega-clight-100 dark:hover:text-vega-cdark-100'
+            )}
+          >
+            <NavbarLinkExternal to={useLinks(DApp.Token)()}>
```

Comment:
Using this component in the top level nav isn't quite right as the OPEN_EXTERNAL icon looks too big next to the dropdown chevron